spread.yaml

```diff
@@ -11,6 +11,37 @@ exclude:
   - .github
 
 backends:
+  # We can switch back to using the official Spread LXD backend after
+  # https://github.com/snapcore/spread/pulls?q=is%3Apr+is%3Aopen+LXD+updated%3A%3C%3D2024-02-23+
+  lxd:
+    type: adhoc
+    allocate: |
+      set -x
+      release=$(echo $SPREAD_SYSTEM | awk -F '-' '{print $2}')
+
+      # Ideally, we would add the ubuntu-minimal remote
+      # e.g. https://cloud-images.ubuntu.com/minimal/releases/jammy/
+      # but that would effectively change the host's LXC configurations.
+
+      echo "Allocating $SPREAD_SYSTEM..."
+      lxc launch --ephemeral ubuntu:$release $SPREAD_SYSTEM
```

Comment:
The runner isn't able to run any `lxc` commands because it is lacking the permissions on lxd unix socket. You can either fix that or run all `lxc` commands with sudo.

Reply:
Where do you see that?

Reply:
Yes, because it needs #317